be1-go/channel/lao/mod.go

```diff
@@ -640,9 +676,12 @@ func (c *Channel) processRollCallClose(msg messagedata.RollCallClose) error {
 		}
 	}
 
+
 	for _, attendee := range msg.Attendees {
 		c.attendees[attendee] = struct{}{}
 
+		createChirpingChannel(attendee, c.channelID, c.hub, c.general, c.log)
```

Comment:
Is the social media linked to an election ? If so, wouldn't it be better to create the chirping channels when a client subscribes to an election ?

Reply:
no it is not linked to an election